app/Http/routes.php

```diff
@@ -38,7 +38,16 @@
 
 /////////// core application ///////////////////
 Route::get('/mothers', 'Admin\CallchampionsController@list_mothers');
+// Route::get('/admins', 'Admin\CallchampionsController@list_admins');
```

Comment:
line no 48 directs to same route?
shall i comment line no 48? and uncomment line no 41